.circleci/config.yml

```diff
@@ -3,12 +3,12 @@ version: 2
 jobs:
   build:
     docker:
-      - image: quay.io/pypa/manylinux1_i686
+      - image: astropy/image-tests-py37-mpl311:1.10
```

Comment:
Is the removal of 32-bit testing deliberate?

Reply:
No, definitely not.  I'm just playing with CircleCI config.  I had assumed `astropy/image-tests-py37-mpl311:1.10` was 32-bit, but included mpl.

I changed the 32-bit tests to include `alldeps`, but the issue I'm having is `mpl` cannot be installed in the `quay.io/pypa/manylinux1_i686` image because of a missing freetype library:
https://app.circleci.com/pipelines/github/astropy/photutils/14/workflows/f4936108-6c07-4455-8001-e1d9087499fe/jobs/1149

Unless you know how to fix this, then I'm going revert to running the 32-bit tests only with the required dependencies (which does not include mpl).

Reply:
Does the rest of this PR look OK?  Everything is ready to be merged, except for this CircleCI issue.

Reply:
There isn't a `mpl` wheel for 32-bit linux, thus CircleCI is compiling from source.